lib/anyway/loaders/ejson.rb

```diff
@@ -23,7 +23,11 @@ def call(name:, ejson_parser: Anyway::EJSONParser.new(EJSON.bin_path), **_option
 
           next unless secrets_hash
 
-          config_hash = secrets_hash[name]
+          config_hash = if ejson_namespace.nil?
```

Comment:
> I think perhaps false would be a better option actually. 

Agree, `false` sounds better. Let's just replace the check with `unless ejson_namespace` to handle both `nil` and `false`